### PR TITLE
customize redirect url after masquerade

### PR DESCRIPTION
**Pull request by GhalebKhaled**

### CODE REVIEW

**Comment by GhalebKhaled** [comment]
[(1) Admin - Enhancement in 'Sign In' in users table [Cycle27]](https://trello.com/c/0MhlmdI8/1556-1-admin-enhancement-in-sign-in-in-users-table-cycle27)

**Review by awartani — APPROVED** [review]

**Comment by GhalebKhaled** [comment]
[DS - Django - Enhancement in 'Sign In' in users table](https://trello.com/c/vCtqh35g/82-ds-django-enhancement-in-sign-in-in-users-table)

File: accountsplus/settings.py [hunk 1]
```diff
@@ -16,6 +17,9 @@ def get_setting(setting_str, is_required, default_value=None):
             return default_value
 
 
+MASQUERADE_REDIRECT_URL = get_setting('MASQUERADE_REDIRECT_URL', False)
```
Comment (alaayameen) [inline accountsplus/settings.py]: @GhalebKhaled no need for this.

File: accountsplus/views.py [hunk 2]
```diff
@@ -84,6 +84,8 @@ def masquerade(request, user_id=None):
         'User {} ({}) masquerading as {} ({})'.format(admin_user.email, admin_user.id, request.user.email, request.user.id))
     django.contrib.messages.success(request, 'Masquerading as user {0}'.format(user.email))
 
+    if settings.MASQUERADE_REDIRECT_URL:
+        return django.http.HttpResponseRedirect(settings.MASQUERADE_REDIRECT_URL)
```
Comment (alaayameen) [inline accountsplus/views.py]: @GhalebKhaled Please use the default login url

**Review by alaayameen — CHANGES_REQUESTED** [review]

**Review by awartani — APPROVED** [review]

**Review by awartani — APPROVED** [review]

**Pull request merged**